bokehjs/src/coffee/mapper/linear_mapper.coffee

```diff
@@ -15,9 +16,17 @@ class LinearMapper extends HasProperties
 
   v_map_to_target: (xs) ->
     [scale, offset] = @get('mapper_state')
-    result = new Float64Array(xs.length)
-    for x, idx in xs
-      result[idx] = scale * x + offset
+    if not _.isNumber(xs[0])
```

Comment:
Nope, sadly. This was one of my big learnings working this up: `_.isArray()` returns `false` for Float64Array's so `_.isArray()` is often not good.